docs/marketplace/dev/feature-walkthroughs/202200.0/marketplace-merchant-custom-prices-feature-walkthrough.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Marketplace Merchant Custom Prices feature walkthrough
+description: Marketplace Merchant Customer Specific Prices lets merchants in Merchant Portal create specific product price for the customer that merchant has relationship with.
+template: feature-walkthrough-template
+---
+
+The *Marketplace Merchant Custom Prices* feature lets merchants in create and edit specific product price for the Customer Business Unit that current merchant has relationship with a help of Merchant Portal UI that is provided.
+
+{% info_block warningBox "User documentation" %}
+
+To learn more about the Marketplace Merchant Custom Prices feature in Merchant Portal and find out how end users use it see Marketplace Merchant Custom Prices feature overview.
+
+{% endinfo_block %}
+
+## Module dependency graph
+
+The following diagram illustrates the dependencies between the modules for the *Marketplace Merchant Custom Prices* feature.
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/5eb0f9ca-ed53-4f7f-a968-38b647712678.png?utm_medium=live&utm_source=custom)
```

Comment:
- this is not Module dependency graph, looks like Domain model, please check other feature walkthrough how it is done
- all diagrams for new release must be placed in https://spryker.atlassian.net/wiki/spaces/DOCS/pages/3305078878/202204.0+Release